ambassador/tests/t_optiontests.py

```diff
@@ -35,7 +49,18 @@ class AddResponseHeaders(OptionTest):
 
     VALUES: ClassVar[Sequence[Dict[str, str]]] = (
         { "foo": "bar" },
-        { "moo": "arf" }
+        { "moo": "arf" },
+        { "zoo": {
+            "append": True,
+            "value": "bar"
+        }},
+        { "xoo": {
+            "append": False,
+            "value": "dwe"
+        }},
+        { "aoo": {
+            "value": "tyu"
+        }}
```

Comment:
The overwriting of headers is not being tested in this. We're simply asserting that the header exists, but not asserting any overwriting. Can you add that please? :)

Reply:
Can you give me a hint on how to set the headers before getting processed in the test, especially multiple headers, I can add the tests accordingly.

Reply:
Have added AddReqHeaderMapping and AddRespHeaderMapping tests based on mapping tests.